### PR TITLE
Bump kinto attachment from 6.2.0 to 6.3.0

### PR DESCRIPTION
**Pull request by grahamalama**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -7,8 +7,16 @@ the version control of each dependency.
...
 
-- Nothing changed yet
+kinto-attachment
+----------------
+
+**kinto-attachment 6.2.0 -> 6.3.0**: https://github.com/Kinto/kinto-attachment/releases/tag/6.3.0
+
+**New features**
+
+- Include the Google Cloud backend automatically when kinto.attachment.gcloud.* settings are used.
```
Comment (leplatrem) [inline CHANGELOG.rst:17]:
```suggestion
- Include the Google Cloud backend automatically when ``kinto.attachment.gcloud.*`` settings are used.
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -7,8 +7,16 @@ the version control of each dependency.
...
 
+Full changelog: https://github.com/Kinto/kinto-attachment/compare/6.2.0...6.3.0
```
Comment (leplatrem) [inline CHANGELOG.rst:19]: I like this, maybe the link could be put in the section title?

**Review by leplatrem — APPROVED** [review]
:ok_hand:

File: CHANGELOG.rst (hunk 2 above)
Comment (grahamalama) [reply CHANGELOG.rst]: Good call!

**Comment by grahamalama** [comment]
@leplatrem addressed your comments in https://github.com/mozilla/remote-settings/pull/214/commits/baae14ebef810372cabad4667f941343faaa1627

**Pull request merged**